src/components/Button.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Button as AntdButton } from 'antd'
+import { BaseButton } from './styles/Button.styled'
+
+type Props = {
+  text: string
+  link?: string
+  fontSize?: string
+}
+
+const defaultProps = {
+  link: 'https://www.google.com',
+  fontSize: '20px',
+}
+function Button(props: Props) {
+  return (
+    <BaseButton fontSize={props.fontSize}>
+      <AntdButton type="text" onClick={() => window.open(`${props.link}`, '_blank')}>
```

Comment:
fix warning below - consider adding a check so if there is no link, then dont do anything onClick

Reply:
7a0053e77d8120f08943acab211b2145bf1b4daf